packages/react-composites/src/composites/CallComposite/pages/CallPage.tsx

```diff
@@ -114,21 +118,37 @@ export const CallPage = (props: CallPageProps): JSX.Element => {
       onRenderGalleryContent={() =>
         _isInCall(callStatus) ? (
           isNetworkHealthy(networkReconnectTileProps.networkReconnectValue) ? (
-            <MediaGallery
-              isMobile={mobileView}
-              {...mediaGalleryProps}
-              {...mediaGalleryHandlers}
-              onFetchAvatarPersonaData={onFetchAvatarPersonaData}
-              /* @conditional-compile-remove(pinned-participants) */
-              remoteVideoTileMenuOptions={options?.remoteVideoTileMenuOptions}
-              drawerMenuHostId={drawerMenuHostId}
-              /* @conditional-compile-remove(click-to-call) */
-              localVideoTileOptions={options?.localVideoTile}
-              /* @conditional-compile-remove(gallery-layouts) */
-              userSetOverflowGalleryPosition={userSetOverflowGalleryPosition}
-              /* @conditional-compile-remove(gallery-layouts) */
-              userSetGalleryLayout={galleryLayout}
-            />
+            dtmfDialerPresent ? (
+              <DtmfDialpadPage
+                mobileView={props.mobileView}
+                modalLayerHostId={props.modalLayerHostId}
+                options={props.options}
+                updateSidePaneRenderer={props.updateSidePaneRenderer}
+                mobileChatTabHeader={props.mobileChatTabHeader}
+                latestErrors={props.latestErrors}
+                onDismissError={props.onDismissError}
+                /* @conditional-compile-remove(capabilities) */
+                capabilitiesChangedNotificationBarProps={props.capabilitiesChangedNotificationBarProps}
+                onSetDialpadPage={() => setDtmfDialerPresent(!dtmfDialerPresent)}
+                dtmfDialerPresent={dtmfDialerPresent}
+              />
```

Comment:
Should we memoize this to avoid rerender when toggling the dialpad button?

Reply:
this will trigger a rerender no matter whether it is memoized or not no? a huge UI change happens when that function fires